TLCGen.Dependencies/TLCGen.Dependencies/Models/Models/Varia/PTPKoppelingModel.cs

```diff
@@ -30,6 +32,33 @@ public PTPKoppelingModel()
             OkBitmapData = new BitmapCoordinatenDataModel();
             ErrorBitmapData = new BitmapCoordinatenDataModel();
             TeKoppelenKruispunt = "";
+            PtpIoIngangen = new List<PtpIoDataModel>();
+            PtpIoUitgangen = new List<PtpIoDataModel>();
+        }
+
+
+        public bool Dummy { get; set; } = false;
+        public bool MaakIO { get; set; } = false;
+
+        [XmlArrayItem(ElementName = "PtpIoIngangen")]
+        public List<PtpIoDataModel> PtpIoIngangen { get; set; }
+
+        [XmlArrayItem(ElementName = "PtpIoUitgangen")]
+        public List<PtpIoDataModel> PtpIoUitgangen { get; set; }
+
+    }
+
+    [Serializable]
+    public class PtpIoDataModel
```

Comment:
Wellicht goed om dit qua naamgeving gelijk te trekken, bv.: PTPKoppelingIoDataModel. Daarnaast goed om de class in een eigen file te plaatsen (Visual heeft daar een quick action voor)

Reply:
Deze en de overige voorstellen / wensen zijn doorgevoerd.